src/pages/index.js

```diff
@@ -90,10 +162,16 @@ function getCardElement(data) {
     cardLikeBtn.classList.toggle("card__like-button_liked");
```

Comment:
```js

  cardLikeBtn.addEventListener("click", () => {
    cardLikeBtn.classList.toggle("card__like-button_liked");
  });
```

Please, delete it

You've set the likes handler in line 154

You need to **toggle likes** only in a `then` block after a successful response from the server because if there is an error, the user will not know about that and will see the toggled heart color.